ext/http/lib.rs

```diff
@@ -101,6 +102,36 @@ impl From<tokio::net::unix::SocketAddr> for HttpSocketAddr {
   }
 }
 
+// implements tower_http::compression's Predicate
+fn not_uncompressible(
```

Comment:
not_uncompressible == is_compressible? double negation is very weird to read IMO

Reply:
Yeah agreed, had fixed this yesterday but hadn't pushed